techniques/developing-localizable-manifests.en.html

```diff
@@ -0,0 +1,216 @@
+          //previousMaturity: "WD",
+          //previousPublishDate:  "2015-04-23",
+		  noRecTrack: true,
+	
+		  
+          group:  "i18n",
+          github: "w3c/i18n-drafts",
+
+    localBiblio:  {
+    "CLDR": {
+	    title:    "Common Locale Data Repository",
+	    href:     "http://cldr.unicode.org",
+        publisher:  "Unicode"
+    },
+    "LDML": {
+	    title:    "Unicode Technical Standard #35: Locale Data Markup Language",
+	    href:     "https://www.unicode.org/reports/tr35/",
+	    publisher: "Unicode",
+	    authors: [
+	       "Mark Davis",
+	       "CLDR Contributors"
+	 ]}
+    }
+
+      };
+
+    </script>
+    <link rel="stylesheet" href="local.css" />
+  </head>
+  <body>
+    <section id="abstract">
+      <p>This document provides definitions and best practices related to the specification of manifest files and similar document formats on the Web.</p>
+    </section>
+    <section id="sotd">
+      <p>The Working Group expects this to become a Working Group Note.</p>
+
+        <p class="note">If you wish to make comments regarding this document, please <a href="https://github.com/w3c/i18n-draffts/issues"
+          style="font-size: 120%;">raise a github issue</a>. You may also send
+          email to the list <a href="mailto:www-international@w3.org?subject=%5Bltli%5D%20">www-international@w3.org</a>
+          (<a href="mailto:www-international@w3.org?subject=subscribe">subscribe</a>,
+          <a href="https://lists.w3.org/Archives/Public/www-international/">archives</a>)
+          as mentioned below. Please include <q>[ltli]</q> at the start of your
+          email's subject. To make it easier to track comments, please raise
+          separate issues or send separate emails for each comment. All comments
+          are welcome.</p>
+
+    </section>
+    <section id="introduction">
+      <h2>Introduction</h2>
+
+        <p>Some specifications on the Web deal with defining sets of files or resources to be consumed together. A common design pattern is to provide a manifest or configuration file that defines how various resources should be used or to provide various kinds of metadata about a collection of resources.</p>
+        
+        <p>Some examples of specifications like this include Webapp [[APPMANIFEST]], Miniapp [[MINIAPP-MANIFEST]], and ePub [[EPUB-33]].</p>
+
+        <p>Manifest files generally contain a mix of <a href="https://w3c.github.io/i18n-glossary/#def_localizable_content">localizable content</a> and <a href="https://w3c.github.io/i18n-glossary/#def_syntactic_content">syntactic</a> or <a href="https://w3c.github.io/i18n-glossary/#def_user_supplied_value">user-supplied values</a> that are not localizable. A specification that defines such a format needs to provide a means for users to localize the manifest into different languages. This document covers the pros and cons of common designs, along with some best practices related to building and managing manifests.</p>      
```

Comment:
Would it be better to change "Manifest files generally" to something like "Some manifest files"?

For example, it seems that there is no human-readable text in [Payment Method Manifest](https://www.w3.org/TR/payment-method-manifest/) or the [origin policy manifest](https://wicg.github.io/origin-policy/#manifest-file).

Reply:
Good callout. See what you think of the updated text.